manifests/cluster-authentication-operator_05_deploy.yaml

```diff
@@ -37,6 +37,8 @@ spec:
           value: quay.io/openshift/origin-hypershift:v4.0
         - name: OPERATOR_IMAGE
           value: docker.io/openshift/origin-cluster-authentication-operator:v4.0  # I believe this is supposed to exist
+        - name: RELEASE_VERSION
```

Comment:
@stlaz another followup: this only covers the operator version.  We need both the operator version and the operand (i.e. the oauth server) version.

Reply:
Also use `OPERATOR_IMAGE` like other folks.